traits/has_traits.py

```diff
@@ -2188,6 +2188,20 @@ def class_editable_traits ( cls ):
         return names
 
     class_editable_traits = classmethod( class_editable_traits )
+    
+    def visible_traits ( self ):
+        """Returns an alphabetically sorted list of the names of non-event
+        trait attributes associated with the current object, that should be GUI visible
+        """
+        return self.trait_names( type = not_event, editable = not_false, visible = not_false )
```

Comment:
According to the use cases I have for traits it doesn't make sense to show non-editable traits by default. With the addition of the visible attribute people can change that behavior anyway.
I guess that your point is that the function name can be somewhat misleading, which I can understand. Would you agree that we change this func name and keep the behavior?